bindsnet/environment/__init__.py

```diff
@@ -126,12 +126,22 @@ def step(self, a: int) -> Tuple[torch.Tensor, float, bool, Dict[Any, Any]]:
             self.update_history()
             self.update_index()
 
-        self.obs = self.encoder(self.obs)
+        raw_obs = self.obs
+
+        # The new standard is BxTxCxHxW. The gym environment doesn't
+        # follow exactly the same protocol.
+        self.obs = self.encoder(self.obs).unsqueeze(1)
```

Comment:
Should this be `unsqueeze(0)`?

Reply:
I've rewritten this part to be a bit more clear